strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryFiles.java

```diff
@@ -156,5 +158,25 @@ public static URI resolveResource(RepositoryPath p)
         
         return relativizeUri(p);
     }
+
+    public static boolean artifactExists(RepositoryPath repositoryPath)
+        throws IOException
+    {
+        return !artifactNotExists(repositoryPath);
+    }
     
+    public static boolean artifactNotExists(RepositoryPath repositoryPath)
```

Comment:
Cosmetic: I think a better sounding name would be `artifactDoesNotExist`. Would you mind changing this?

Reply:
done